cfme/tests/infrastructure/test_vm_rest.py

```diff
@@ -135,3 +137,48 @@ def test_delete_vm_from_collection(vm):
         casecomponent: Infra
     """
     delete_resources_from_collection([vm], not_found=True, num_sec=300, delay=10)
+
+
+@pytest.fixture
+def vm_ip(appliance, provider):
+    vm = appliance.collections.infra_vms.instantiate(
+        provider.data["cap_and_util"]["capandu_vm"], provider
+    )
+    view = navigate_to(vm, "Details")
+    rest_vm = appliance.rest_api.collections.vms.get(name=vm.name)
+    # get all the ipaddresses from VM Details page
+    ipaddrs = (
```

Comment:
optional: I think better to add this as `property` in model page with will return list of assignee ips.

Reply:
Hmm, I was thinking the same, but then I thought it won't be useful to others so I didn't add one. If you think it'll be helpful to everyone, sure I'll add one! :)

Reply:
Thanks for this suggestion @digitronik, I realized I had written plenty of code that was really not necessary!

Reply:
same code need to move in model; they why it can not be part of model?